src/pages/episodes.js

```diff
@@ -0,0 +1,25 @@
+import {useEffect} from 'react';
+
+import DisplayEpisodes from '../components/DisplayEpisodes';
+import useStore from '../hooks/useStore';
+
+export default function Episodes() {
+	const podcasts = useStore(state => state.podcasts);
+	const selectedEpisodes = useStore(state => state.selectedEpisodes);
+	const setSelectedEpisodes = useStore(state => state.setSelectedEpisodes);
+
+	//hardcoded podcast selection, because there are none to choose from yet
+	const selectedPodcast = podcasts.find(
+		podcast => podcast.name === 'Syntax - Tasty Web Development Treats'
+	);
+
+	useEffect(() => {
+		setSelectedEpisodes(selectedPodcast.key);
+	}, []);
```

Comment:
You don't need a `useEffect` hook here. `useEffect` hooks are only used for side effects in React (f.e. a fetch or local storage). It should work just the same without the `useEffect`.

Reply:
Okay. My idea was to employ useEffect to prevent multiple execution.

Reply:
And actually I think I might be on to something with this. When I remove `useEffect`, the browser tab kind of crashes and I get a warning that my app slows Firefox down.

Reply:
Alright, I didn't test it, it just looked as if it would work without :)